cmd/local.go

```diff
@@ -406,9 +406,9 @@ func local(cmd *cobra.Command, args []string) {
 	}
 }
 
-//WriteGitIgnoreFile takes a provided path and does best attempt work to write a "Exclude all" gitignore file in the location
+//WriteGitIgnoreFile takes a provided path and does best attempt work to write a "Exclude all" gitignore file in the location.
 func WriteGitIgnoreFile(filepath string) {
-	utils.WriteLines([]string{"*"}, path.Join(filepath, ".gitignore"))
+	errpanic(utils.WriteLines([]string{"*"}, path.Join(filepath, ".gitignore")))
```

Comment:
This is a scenario where it would not be moot since WriteGitIgnore is in the execution path. It's part of the setup phase for making sure random nonmem files don't get added to git in the output structures. These are the pathways we have to worry about the panics in. 

The other ones mentioned are in fact during startup, but using it all over the place could have the unexpected consequence of killing the ability to fire off postexecution scripts. Their main purpose was to notify a user whether a job succeeded or failed

Reply:
So you have a situation where:

1. a function touches the filesystem
2. the functions in the filesystem can fail, leaving everything in a weird state
3. the function does not return error
4. you don't want to throw away your data

what do we do in these situations?

Reply:
In a scenario where the function doesn't return an error, it needs to get surfaced to the execution phases (the turnstile interface commands). If the error gets surfaced there, the manager will at least log to its std out that an error occurred. 

In an SGE environment the entire output of the bbi run is stored in a log file. Secondarily if the job was not successful (ie collected an error) if a user is utilizing the post-execution script hooks, it would indicate whether or not that run was successful. The goal would just be to elevate it to the manager so that a ConcurrentError can be recorded.